coralproject/talk
# Install Talk

# Install Talk

## What does this PR do?
Adds:

- New Coral-UI Components
	* Wizard
	* WizardNav
	* Select
	* Option
- New `/setup` endpoint and cli setup support	
* Install Route `/admin/install`


## How do I test this PR?
- Drop your db. (Your settings should not have been initialized) !important;
- Go to `/admin/install`
* You should see a wizard install
* Do the install
